etc/scripts/install-on-minikube.sh

```diff
@@ -31,7 +31,17 @@ minikube config set cpus ${CPUS}
 minikube config set disk-size ${DISK_SIZE}
 
 # Start minikube
-minikube start -p knative --extra-config=apiserver.enable-admission-plugins="LimitRanger,NamespaceExists,NamespaceLifecycle,ResourceQuota,ServiceAccount,DefaultStorageClass,MutatingAdmissionWebhook"
 
-DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
-"$DIR/install.sh" -q
+if [-z "$VM_DRIVER"]; then
+  minikube start -p knative --extra-config=apiserver.enable-admission-plugins="LimitRanger,NamespaceExists,NamespaceLifecycle,ResourceQuota,ServiceAccount,DefaultStorageClass,MutatingAdmissionWebhook"
+else
+  minikube start -p knative --vm-driver $VM_DRIVER --extra-config=apiserver.enable-admission-plugins="LimitRanger,NamespaceExists,NamespaceLifecycle,ResourceQuota,ServiceAccount,DefaultStorageClass,MutatingAdmissionWebhook"
```

Comment:
I would still prefer to remove this conditional, and set `VM_DRIVER=${VM_DRIVER:-virtualbox}` with the other env vars in the script. It gives the users a single place to see what options they can configure, as well as their defaults. That way, you only need line 38.

Reply:
@jcrossley3 : Make sense. Fixed.